### DISCUSSION TITLE
At Glance - go backward / go forward

### DISCUSSION THREAD

**Discussion by 0skar2** [zen-browser/desktop#2719] [Ideas]
### Captchas

- [ ] I have read the instructions.
- [ ] I have searched existing issues and avoided creating duplicates.
- [ ] I am not filing an enhancement request.

### What happened?

Since at glance supposedly behaves like a tab, and i can explore other pages inside of it, i'd also expect the ability to go backwards, or forwards, like in case of normal tabs, thanks

### Reproducible?

- [ ] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a.17

### What platform are you seeing the problem on?

Linux

**Comment by RayZ3R0** [comment]
You can with shortcuts or mouse side buttons
